examples/kmlParse.html

```diff
@@ -0,0 +1,227 @@
+    <link rel="stylesheet" href="static/examples.css">
+    <script src="static/codemirror-compressed.js"></script>
+    <script src="static/helpers.js"></script>
+    <script src="../Cesium/Cesium.js"></script>
+    <script src="static/jquery-2.1.3.min.js"></script>
+    <script src="static/bootstrap-3.3.2-dist/js/bootstrap.min.js"></script>
+</head>
+
+<body onresize="onResize()">
+<header class="container">
+    <div class="navbar navbar-default navbar-fixed-top">
+        <div class="navbar-header">
+            <button class="btn btn-primary navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                <span class="glyphicon glyphicon-menu-hamburger"></span>
+            </button>
+            <a class="navbar-brand" href="http://cesiumjs.org" target="_blank"><img src="static/Cesium_Logo_forDarkBackground.png" /></a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+            <ul class="nav navbar-nav">
+                <li><a href="#" id="menuPreview" onclick="showPanel(this)">Show Preview</a></li>
+                <li><a href="#" id="menuCesium" onclick="showPanel(this)">Show Cesium Code</a></li>
+                <li><a href="#" id="menuGoogle" onclick="showPanel(this)">Show Google Earth Code</a></li>
+                <li><a href="http://cesiumjs.org/for-google-earth-developers.html" target="_blank">For Google Earth Developers</a></li>
+                <li><a href="../index.html" target="_blank">All Google Earth examples</a></li>
+                <li class="dropdown">
+                    <a href="#" data-toggle="dropdown" class="dropdown-toggle">Additional Resources <b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li><a href="http://cesiumjs.org/tutorials.html" target="_blank">Tutorials</a></li>
+                        <li><a href="http://cesiumjs.org/Cesium/Apps/Sandcastle/index.html" target="_blank">Cesium Sandcastle</a></li>
+                        <li> <a href="http://cesiumjs.org/refdoc.html" target="_blank">Reference Documentation</a> </li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+    </div>
+</header>
+<div id="preview">
+    <div id="previewContent">
+        <div id="cesiumContainer"></div>
+    </div>
+</div>
+<div id="code">
+    <div class="panel panel-primary" id="czmPanel">
+        <div class="panel-heading">Relevant Cesium Source</div>
+        <div class="panel-body">
+            <textarea id="cesiumSource"></textarea>
+        </div>
+    </div>
+    <div class="panel panel-primary" id="gePanel">
+        <div class="panel-heading">Relevant Google Earth Source (<a target="_blank" href="https://code.google.com/p/earth-api-samples/source/browse/trunk/examples/hittest.html">link</a>)</div>
+        <div class="panel-body">
+                <textarea id="gesource">
+                    // Sample KML taken from
```

Comment:
Was this KML already in the original code?  If so, it should be covered by the Apache 2.0 license and OK to use.  If it was just on their website and we added it to the code, we'll need to find or create another KML example.